hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/ListInfoSubcommand.java

```diff
@@ -51,6 +51,19 @@
       defaultValue = "")
   private String uuid;
 
+  @CommandLine.Option(names = {"--NodeOperationalState"},
+      description = "Show info by datanode NodeOperationalState." +
+          "IN_SERVICE DECOMMISSIONING DECOMMISSIONED ENTERING_MAINTENANCE" +
```

Comment:
Please change the description to the following format "Show info by datanode NodeOperationalState(IN_SERVICE, DECOMMISSIONING, DECOMMISSIONED, ENTERING_MAINTENANCE, IN_MAINTENANCE)."

Reply:
ok